repour/adjust/project_manipulator_provider.py

```diff
@@ -100,12 +101,12 @@ async def adjust(work_dir, extra_adjust_parameters, adjust_result):
                 default_parameters,
             )
             temp_suffix = ("-" + orig_inc_suffix) if orig_inc_suffix else ""
-            temp_build_parameters.append(
+            alignment_parameters.append(
                 "-DversionIncrementalSuffix=" + suffix_prefix + temp_suffix
             )
 
-        if specific_indy_group:
-            temp_build_parameters.append("-DrepositoryGroup=" + specific_indy_group)
+        if brew_pull_enabled:
```

Comment:
This should not be here for Project Manipulator. NPM packages cannot be pulled from Brew, they are from PNC only